site/src/atomics/ratio.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Ratio
+description: Aspect ratio related atomic css classes for the Atlas Design System
+template: standard
+classType: Atomics
+classPrefixes:
+  - aspect-ratio
+---
+
+# Ratio Atomics
+
+Ratio atomics are used to add a fluid aspect ratio to an element. The element will scale to fill 100% width of the container while matching the height to the defined aspect ratio.
+
+| modifiers | ratio                          |
+| --------- | ------------------------------ |
+| values    | `16-9`,`9-16`, `4-3`, `square` |
+
+## Usage
+
+Here are examples of ratio atomics, used on `iframe`, `video` and `img` elements. When used on a `video` element or `iframe` video with the matching aspect ratio, it will remove the black bars and scale responsively. When used on an `img` of a different aspect ratio it will crop the image.
+
+```html
+  <div class="padding-sm width-500-desktop">
+  	<video controls name="media" class="width-500 aspect-ratio-16-9" poster="~/src/scaffold/media/sample-learn-video-thumbnail.jpg" src="~/src/scaffold/media/sample-learn-video.mp4" type="video/mp4" />
+  	</video>
+  </div>
+```
+
+```html
+<div class="width-250 padding-sm">
+	<iframe
+		class="aspect-ratio-9-16"
+		src="~/src/scaffold/media/sample-learn-vertical-video.mp4"
```

Comment:
This video autoplays when I hit the page, audio and all, which is kind of startling. Is there a way we can avoid that?

Reply:
Weird. Can't seem to reproduce that now 🤔

Reply:
Yea I pushed a fix for that using `sandbox` attribute on the iframe.